src/python/bench/test_hf_raw.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
```

Comment:
Consider adding a short comment at the top of this file what this script does and a usage example.